src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard/MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = { movies: Movie[] };
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div>
```

Comment:
you have missed the class name, that's why in your demo cards looks bigger than it should be <div className="movies-list"> ;
and also try not to write everything in one line)

Reply:
Thanks!

Just fix it)